### PR TITLE
Update CarTurrets.cs

### PR DESCRIPTION
**Pull request by psycodeliccircus**
Adding translation in portuguese brazil

### CODE REVIEW

File: CarTurrets.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 namespace Oxide.Plugins
 {
-    [Info("Modular Car Turrets", "WhiteThunder", "1.4.0")]
+    [Info("Modular Car Turrets", "WhiteThunder", "1.4.1")]
```
Comment (WheteThunger) [inline CarTurrets.cs]: Can you undo this change? Having the version change will trigger update notifications to users, which I prefer to avoid for updates that only provide translations.

**Review by WheteThunger — CHANGES_REQUESTED** [review]

File: CarTurrets.cs (hunk 1 above)
Comment (psycodeliccircus) [reply CarTurrets.cs]: ok

Comment (psycodeliccircus) [reply CarTurrets.cs]: The version change is already done @WheteThunger

File: CarTurrets.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿using Newtonsoft.Json;
+using Newtonsoft.Json;
```
Comment (WheteThunger) [inline CarTurrets.cs:1]: Why is this line showing as changed?
